pkg/services/ngalert/models/instance.go

```diff
@@ -44,8 +44,12 @@ func (i InstanceStateType) IsValid() bool {
 		i == InstanceStateError
 }
 
+type SaveAlertInstancesCommand struct {
+	Instances []SaveAlertInstanceCommandFields
+}
+
 // SaveAlertInstanceCommand is the query for saving a new alert instance.
-type SaveAlertInstanceCommand struct {
+type SaveAlertInstanceCommandFields struct {
```

Comment:
```suggestion
type SaveAlertInstanceCommand struct {
```

Reply:
I think it makes sense to keep this struct called `SaveAlertInstanceCommand` and then the other struct either `SaveAlertInstancesBulkCommand` or `SaveAlertInstancesBatchCommand`?

Reply:
I ended up making the function variadic, so this is moot.